client/packages/electron/src/electron.ts

```diff
@@ -25,28 +28,42 @@ if (require('electron-squirrel-startup')) {
   app.quit();
 }
 
+const connectToServer = (window: BrowserWindow, server: FrontEndHost) => {
+  isDiscovering = false;
+  const url = frontEndHostUrl(server);
+  // Can change 'frontEndHostUrl(server)' to http://localhost:3003 if you want client to display front end served by webpack
+  window.loadURL(url);
+  connectedServer = server;
+  systemPreferences.setUserDefault('/mru/server', 'string', url);
+};
+
 const start = (): void => {
   // Create the browser window.
   const window = new BrowserWindow({
-    height: 600,
-    width: 800,
+    height: 768,
+    width: 1024,
     webPreferences: {
       preload: MAIN_WINDOW_PRELOAD_WEBPACK_ENTRY,
     },
   });
   // and load the index.html of the app.
   window.loadURL(MAIN_WINDOW_WEBPACK_ENTRY);
 
+  // allow the user to press Alt and force display of the server selection
+  window.webContents.on('before-input-event', (event, input) => {
+    if (input.alt) {
```

Comment:
this was annoying.. I used `localStorage` initially and that was not being retained : re-opening the app clears localStorage for the discovery screen. The language is retained, so localStorage is ok within the main app